TicTacToe2/TicTacToeGame.cs

```diff
@@ -6,13 +6,15 @@ namespace TicTacToe2
 {
     class TicTacToeGame
     {
+        static readonly string[] space = new string[9] {"1", "2", "3", "4", "5", "6", "7", "8", "9"};
```

Comment:
A few things here

1. The static keyword makes is so this variable is shared across all objects of a class. This means that if two TicTacToeGames are created, they would both use the same spaces. It's preferable that each game has its own unique spaces, so the static keyword should be removed
2. This variable is a member of a class. The non-private members a class exposes compose its interface - a set of actions that can be taken on that class. If this space array is exposed, the ability to change the values in the array becomes part of the class's interface. We don't want that, so this member should be private. See encapsulation (https://www.tutorialspoint.com/csharp/csharp_encapsulation.htm)
3. There should be one blank line between member variables and functions